### PR TITLE
Point to latest webservice build.

### PR DESCRIPTION
**Pull request by coverbeck**

### CODE REVIEW

**Comment by garyluu** [comment]
Change my semaphore file while you're at it

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
+          - ng lint
+          - npm run build.prod
+          - npm install codecov
+          - ng test --watch=false --code-coverage --browsers ChromeHeadless
+          - ./node_modules/codecov/bin/codecov
+  - name: "Integration Tests"
+    task:
+      env_vars:
+        - name: WEBSERVICE_VERSION
+          value: 1.6.0-alpha.7
```
Comment (garyluu) [inline .semaphore/semaphore.yml]: Here too, thanks :smile:

**Comment by denis-yuen** [comment]
@coverbeck updating semaphore is really not necessary until we use it more broadly across the team

**Comment by garyluu** [comment]
Takes very little time to update the semaphore.yml so that I don't get any unexpected surprises and have to update it myself after it fails

**Comment by denis-yuen** [comment]
nevertheless, I'd like to consider updating these as a favour that cannot block PRs 
sorry @garyluu

**Comment by coverbeck** [comment]
@denis-yuen Saw your comment after I'd started the change to semaphore.

**Comment by garyluu** [comment]
Ok, I will make my own commits alongside these when I see it since it actually is necessary for my work.

**Pull request merged**